### ISSUE TITLE
Bug: VerbGuac will not start because currentTheme.txt does not exist

### ISSUE THREAD

**Issue by bmullan** [benjaminbhollon/verbose-guacamole#15]
I read the Documentation here:

**[Installation on Ubuntu](https://verboseguacamole.com/install/Ubuntu/)**

Says...

>  First, find the most recent version on the Github Releases page. **Don't download a version that's title includes "Beta" or "RC", those are preview releases and not intended for normal use.**

However, going to the Releases Assets page **ALL** of them are marked **_Pre-Release_** ?

I did install **[v0.3.4](https://github.com/benjaminbhollon/verbose-guacamole/releases/tag/v0.3.4)**
on an Ubuntu 20.04.3 desktop system but when I try to execute, it looks like its trying to start but gives up !

Is there are way to get a working version on Ubuntu 20.04 ?

thanks
brian

**Comment by benjaminbhollon** [comment]
Ha, okay. So that site is actually still under development and I wasn't expecting anyone to find it this early! I just started it last week. Also, the full program is not yet ready for release (I'm hoping to finish in the next week), so yes they're all pre-releases. I should probably add a note to the documentation until then.

I'm not sure why it wouldn't run on Ubuntu 20.04, but I can check that out. I used the Ubuntu 21.04 to build it, so maybe there are some inconsistencies. Have you tried installing via the command line? (`sudo apt install ./<filename>`)? I've found that works for some programs when the graphical installer doesn't.

There might also be an issue with the git native dependency, so make sure you have git installed. I thought I implemented code to warn you if you don't have it, but it might not be working.

**Comment by benjaminbhollon** [comment]
Ah, I just realized you were saying it fails when you try to run the program, not on installation.

I just tried it myself and it works fine on Ubuntu 21.10 Beta (I'm currently using it because I love a lot of things about GNOME 40 and Ubuntu fixes the things I didn't like with it on other distros), I'll make a VM with 20.04 on it to test.

I went ahead and updated the site with a notice warning that there are currently only pre-releases.

**Comment by bmullan** [comment]
cool Ben... thx

**Comment by benjaminbhollon** [comment]
Well, I tested it and you're right, it didn't run. The issue was that during development I had created a config file that wasn't present on a clean install, and it threw an error looking for it.

![image](https://user-images.githubusercontent.com/33023252/136685615-9b1877e3-1d4e-4332-ae5c-08b09d417f08.png)

You should be able to fix it by running the following three commands:

```bash
mkdir ~/.config/verbose-guacamole/
touch ~/.config/verbose-guacamole/currentTheme.txt
echo "guacamole" >> ~/.config/verbose-guacamole/currentTheme.txt
```
To make sure it worked run the following command and make sure it returns "guacamole" (no quotes):

```bash
cat ~/.config/verbose-guacamole/currentTheme.txt
```

While I can't go back and fix this in the v0.3.4 release, I'll add the workaround to the release page and will make sure I fix it in the v0.3.5 release. Thanks for catching this, @bmullan!

**Comment by benjaminbhollon** [comment]
Ha, looks like I'd already fixed this since the last release, though I don't remember doing it. Well, either way, it should be fixed on v0.3.5.
